tgui/public/tgui.html

```diff
@@ -45,8 +45,8 @@
       : null;
   })();
   // Basic checks to detect whether this page runs in BYOND
-  var isByond = tridentVersion !== null
-    && location.hostname === '127.0.0.1'
+  var isByond = /*tridentVersion !== null
+    &&*/ location.hostname === '127.0.0.1'
```

Comment:
and this

this was for chromium shenanigans

Reply:
Improve detection by checking for window.cef_to_byond here.